web/themes/custom/cloudy/src/css/_bootstrap-overrides.scss

```diff
@@ -1,42 +1,23 @@
 /* Colors */
 
 // Assigns color names to specific hex values
-$blue: #418fde;
-$blue-dark: #1b5a99;
-$blue-darker: #0b4177;
-$blue-light: #e8eff5;
-$cyan: #17a2b8;
-$green: #046a38;
-$green-light: #e7f4e4;
-$red: #dc3545;
-$yellow: #ffb81c;
-$light-gray: #e8eff5;
-$gray: #e8eff5;
-$white: #fff;
-$black: #000;
+$blue: $cloudy-blue-base;
+$cyan: $cloudy-cyan-base;
+$green: $cloudy-green-base;
+$red: $cloudy-red-base;
+$yellow: $cloudy-yellow-base;
+$gray: $cloudy-gray-base;
+$white: $cloudy-white-base;
+$black: $cloudy-black-base;
 
 // Assigns primary, secondary, light, and dark to specific colors
-$primary: $blue-dark;
-$secondary: $blue-dark;
+$primary: $cloudy-blue-dark;
+$secondary: $cloudy-blue-dark;
 $light: $gray;
 
-// Reassign color vars to semantic color scheme
-$brand-primary: $primary;
-$brand-secondary: $secondary;
-
-// Alerts colors
-$primary-alt: $primary;
-$primary-alt-lightest: $blue-light;
-$gold: $yellow;
-$gold-lightest: #fff1d2;
-$secondary-alt: $red;
-$secondary-alt-lightest: #f9dede;
-$green-alt: $green;
-$green-alt-lightest: $green-light;
-
 // Anchor elements
-$link-color: $blue-dark;
-$link-hover-color: darken($link-color, 5%);
+$link-color: $cloudy-blue-dark;
+$link-color-hover: darken($link-color, 5%);
```

Comment:
$link-hover-color is a variable in bootstrap that was being overwritten. Renaming it here removes the override to that variable. Please revert.